pkgs/palantir/conjure-rust/pkg.yaml

```diff
@@ -0,0 +1 @@
+packages: []
```

Comment:
Test data is empty.
We need to fix this.

Reply:
What needs to go in here?